operator/src/test/java/org/keycloak/operator/ClusterOperatorTest.java

```diff
@@ -122,6 +137,41 @@ private static void calculateNamespace() {
     namespace = "keycloak-test-" + UUID.randomUUID();
   }
 
+  protected static void deployDB() {
+    // DB
+    Log.info("Creating new PostgreSQL deployment");
+    k8sclient.load(KeycloakDeploymentE2EIT.class.getResourceAsStream("/example-postgres.yaml")).inNamespace(namespace).createOrReplace();
+
+    // Check DB has deployed and ready
+    Log.info("Checking Postgres is running");
+    Awaitility.await()
```

Comment:
This is almost a note to self (no changes needed).
Waiting for the DB to be up get our tests in a "stable" situation, would it be better to instead "just deploy everything at once" and see if the operator is robust enough to catch up?

Reply:
deal 👍